example/main.jsonnet

```diff
@@ -0,0 +1,52 @@
+local obs = (import '../environments/base/observatorium.jsonnet');
+
+{
+  'observatorium-crd': {
```

Comment:
I don’t think this is even necessary just wrote the cr directly

Reply:
I hope this time, I understand you correctly :)